src/item/views/ItemDetail.tsx

```diff
@@ -786,6 +792,16 @@ const ItemDetail: FunctionComponent<ItemDetailProps & DefaultSecureRouteProps<{
 							}}
 						/>
 					)}
+					{PermissionService.hasPerm(user, PermissionName.VIEW_ITEMS_OVERVIEW) && (
+						<Button
+							type="tertiary"
+							icon={IconName.edit}
+							label={tText('item/views/item-detail___media-item-beheren')}
+							ariaLabel={tText('item/views/item-detail___media-item-beheren')}
+							title={tText('item/views/item-detail___media-item-beheren')}
+							onClick={navigateToItemDetail}
+						/>
```

Comment:
Use a link around a button if you have a specific url you want to navigate to. Since it allows opening in a new tab while a button does not allow that

eg:
```
<Link to={editLink}>
	<Button
		type="secondary"
		icon={IconName.edit}
		ariaLabel={}
		title={}
	/>
</Link>
```